src/check.ts

```diff
@@ -59,6 +59,21 @@ export interface CheckRequest {
   externalContent?: ExternalContent;
 }
 
+export interface LiveSearchRequest {
+  'request-id': string;
```

Comment:
I am interested to know why the 'request-id' is in single quotes.

Reply:
Added quotes only for the hyphen.

Reply:
@loaiabdelhalim does this answer your question ?